vowpalwabbit/core/src/reductions/oaa.cc

```diff
@@ -34,10 +34,10 @@ struct oaa
   uint64_t num_subsample = 0;           // for randomized subsampling, how many negatives to draw?
   uint32_t* subsample_order = nullptr;  // for randomized subsampling, in what order should we touch classes
   size_t subsample_id = 0;              // for randomized subsampling, where do we live in the list
-  int indexing = -1;                    // for 0 or 1 indexing
   VW::io::logger logger;
+  uint32_t& indexing;  // for 0 or 1 indexing
```

Comment:
can you add a comment for indexing == 2 or put it as a constant somewhere with a clear name?